docs/application/native/guides/device/system.md

```diff
@@ -468,6 +468,7 @@ The following table lists the network feature keys:
 | `http://tizen.org/feature/network.wifi.direct.display` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/network.wifi` key, if the device supports Wi-Fi Direct display feature. |
 | `http://tizen.org/feature/network.wifi.direct.service_discovery` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/network.wifi` key, if the device supports Wi-Fi Direct service discovery. |
 | `http://tizen.org/feature/network.wifi.tdls` | `bool` | The platform returns `true` for this key, if the device supports the Wi-Fi Tunneled Direct Link Setup (TDLS). |
+| `http://tizen.org/feature/network.inm` | `bool` | The platform returns `true` for this key, if the device has supports the Intelligent Network Monitoring (INM) feature. |
```

Comment:
@suraj-geddam:

Kindly remove ‘has’ in this sentence, for consistency with other sentences on the page:

The platform returns `true` for this key, if the device supports the Intelligent Network Monitoring (INM) feature.